opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -995,8 +995,18 @@ def record_exception(
                 type(exception), value=exception, tb=exception.__traceback__
             )
         )
+        module = (
```

Comment:
i think using a single if would make it more readable instead of doing two inline ifs.What do you think?

Reply:
Agreed. Let me improve it